.github/workflows/api-qa.yml

```diff
@@ -24,12 +23,3 @@ jobs:
       OAUTH2_CLIENT_ID: ${{ secrets.DEV_MOBILITY_FEEDS_OAUTH2_CLIENT_ID}}
       OAUTH2_CLIENT_SECRET: ${{ secrets.DEV_MOBILITY_FEEDS_OAUTH2_CLIENT_SECRET}}
 
-  integration-tests:
-    uses: ./.github/workflows/integration-tests.yml
-    needs:
-      - api-build-deployment
-    with:
-      API_URL: 'https://api-qa.mobilitydatabase.org'
-    secrets:
-      REFRESH_TOKEN: ${{ secrets.QA_API_TEST_REFRESH_TOKEN }}
-
```

Comment:
Removing the integration test here as the caller workflow will be executing integration tests after API deployment.